websites/W/Wolvesville/presence.ts

```diff
@@ -336,17 +346,64 @@ presence.on("UpdateData", async () => {
 		//Game
 	} else if (document.location.href.includes("wolvesville.com")) {
 		const root = document.querySelector("#root");
+		let status: HTMLDivElement;
 
-		document.querySelector(
-			"div.css-1dbjc4n.r-1p0dtai.r-18u37iz.r-1777fci.r-1d2f490.r-98ikmy.r-u8s1d.r-zchlnj > div.css-1dbjc4n.r-1ffj0ar.r-z2wwpe.r-18u37iz.r-1w6e6rj.r-1777fci.r-1l7z4oj.r-gu0qjt.r-85oauj.r-95jzfe > div.css-1dbjc4n.r-1awozwy.r-1777fci > div.css-1dbjc4n.r-17bb2tj.r-1muvv40.r-127358a.r-1ldzwu0.r-z80fyv.r-19wmn03"
-		) && (presenceData.details = "Loading Wolvesville...");
+		//Get player status
+		if (
+			document.querySelector(
+				"div.css-1dbjc4n.r-18u37iz > div.css-1dbjc4n.r-173mn98.r-19u6a5r.r-12zb1j4.r-1inuy60"
+			) ||
+			(document.querySelector(
+				"div.css-1dbjc4n.r-1awozwy.r-1p0dtai.r-1777fci.r-1d2f490.r-u8s1d.r-zchlnj.r-ipm5af"
+			) &&
```

Comment:
Why are there extra brackets?

Reply:
Because it’s an AND statement with an OR behind it